.github/workflows/test-pr.yml

```diff
@@ -102,6 +102,162 @@ jobs:
           if-no-files-found: error
           retention-days: 1
 
+  pyk-code-quality-checks:
```

Comment:
Would it make sense to put these into a separate workflow? (If I know correctly triggers are per-workflow.)

In that case of course, the Debian package would have to be shared between the workflows.